content/1_General/USACO_FAQs.mdx

```diff
@@ -238,6 +238,73 @@ or by googling. Answers to additional FAQs can be found
 You can also ask a question by submitting the "Contact Us" form on the
 bottom-left of this page, or on the [USACO Forum](https://forum.usaco.guide/).
 
+
+## Q: How can I register or create an account?
+
+To register for USACO, visit the official website (www.usaco.org) and look for the registration or signup link. If applicable, follow the prompts to create your account by providing the necessary information such as your name, email address, and possibly your school information. Once registered, you'll have access to participate in USACO contests.
+
+## Q: What additional steps should I take to get ready?
+
+Besides registering for an account, it's advisable to prepare by practicing programming and problem-solving skills. You can explore past USACO problems available on their website to familiarize yourself with the types of challenges you'll encounter. Additionally, consider studying algorithms and data structures relevant to the competition.
+
+## Q: How do I initiate or begin a contest?
+
+Once you've registered and logged into your USACO account, check the website during contest weekends for instructions on starting the contest. Usually, there will be a button or link on the homepage indicating the start of the contest. Click on it to initiate your personal contest timer and access the contest problems.
+
+## Q: Is collaborative work allowed?
+
+Collaborating with others during the contest is strictly prohibited to maintain fairness and integrity. USACO contests are designed to assess individual problem-solving skills, and allowing collaboration could provide an unfair advantage. However, outside of the contest environment, students are encouraged to study and prepare together, sharing knowledge and techniques to improve their skills collectively.
+
+## Q: Is participation on a school team mandatory?
+
+No, participating on a school team is not mandatory for USACO contests. Competitors can participate as individuals, and teamwork during the contest itself is not allowed.
+
+## Q: Are there any awards or recognitions for participation?
+
+While USACO does not offer physical prizes, advancing through the difficult divisions and achieving high scores is an accomplishment worth noting. Recognition is given by listing the names of advancing participants in the contest results, which can be a notable achievement to include on college applications and resumes. Additionally, the skills and knowledge gained through participation can be valuable assets in academic and professional pursuits.
+
+## Q: What is the scoring system for contests?
+
+USACO contests are scored out of 1000 points, with each problem weighted equally. Solutions are evaluated against a set of test cases, and points are awarded based on the number of correct test cases passed.
+
+## Q: Are there any penalties for performing poorly?
```

Comment:
well, you could get discouraged and stop participating in contests ...

(I don't think this answer is useful, remove?)